src/mqt/bench/utils.py

```diff
@@ -345,7 +345,7 @@ def create_zip_file():
     return subprocess.call(f"zip -rj {get_zip_file_path()} {get_qasm_output_path()}", shell=True)
 
 
-def calc_qubit_index(qargs: list[Any], qregs: list[QuantumRegister], index: int) -> Any:
+def calc_qubit_index(qargs: list[Any], qregs: list[QuantumRegister], index: int) -> int:
```

Comment:
Coming back to this again: Why is the `qargs` argument typed as `list[Any]` instead of a proper type?

Reply:
tbh, because I have overlooked that. Added it by now.